docs/install.md

```diff
@@ -0,0 +1,57 @@
+# fn install
+
+This command will help you to install OpenFunction and its dependencies.
+
+## Parameters
+
+```shell
+--all              For installing all dependencies.
+--async            For installing OpenFunction Async Runtime(Dapr & Keda).
+--cert-manager     For installing Cert Manager.
+--dapr             For installing Dapr.
+--dry-run          Used to prompt for the components and their versions to be installed by the current command.
+--ingress          For installing Ingress Nginx.
+--keda             For installing Keda.
+--knative          For installing Knative Serving (with Kourier as default gateway).
+--region-cn        For users in China to speed up the download process of dependent components.
+--shipwright       For installing ShipWright.
+--sync             For installing OpenFunction Sync Runtime(Knative).
```

Comment:
--sync means OpenFunctionSync runtime and probably needs to install [keda-http-addon](https://github.com/kedacore/http-add-on/blob/main/docs/install.md) , --knative is a seperate runtime